code-challenges.test.js

```diff
@@ -32,6 +76,17 @@ var num3 = -7
 
 // b) Create the function that makes the test pass.
 
+//write a function called evenlyDivisble that takes in an argument of a number 
+const evenlyDivisible = (myNums) => {
+    //if myNums is evenly divisible by 3
+    if(myNums % 3 === 0) {
+        //return `${myNums} is divisible by three`
+        return `${myNums} is divisible by three`
+    } else if(myNums % 3 !== 0){
```

Comment:
Because the conditionals `myNums % 3 === 0` and `myNums % 3 !== 0` are exact opposites, it's safe to use an else block instead of an else if block